docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
+
+Quay.io does not maintain a list of official images like Docker Hub, but you can nonetheless find official images on there with a bit of searching, such as `CentOS <https://quay.io/repository/centos/centos?tab=info>`__. Because Quay is owned by Red Hat, it is a good place to find images based on Red Hat Linux distributions.
+
+Looking for an oft-maintained image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Official images are generally well-maintained. By this we mean that they get frequent updates. Most Docker registries will show you the last time an image was updated, and will show a history going back several updates. This is important not only because it means you can get the latest features of new versions of software, but it also usually indicates that someone is paying attention to the security side of things too. No one wants to use a container only to discover that it still contains `the infamous log4j vulnerability <https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046>`__, after all.
+
+Tutorial: Downloading and running an existing Docker image
+----------------------------------------------------------
+
+Finding an official image
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Let's start with Docker Hub. From the homepage, we can click "Explore" in the top righthand corner in order to get to `the search page <https://hub.docker.com/search?q=>`__, allowing us to start digging through all of its images. Don't worry, it won't load all 9,200,012 images at once.
+
+First, let's click the "Docker Official Image" option on the left, then search "Python" using the search bar at the top. Fittingly, our top result is the official Python image.
+
+.. image:: /assets/images/docs/docker/dockerhub-search-python.png
+    :alt: Screenshot of the Docker Hub search feature, with the official Python image showing up as the first result
+
+Upon clicking on that result, we are taken to a description page. There is a lot of information here, such as an explanation of the multiple variants of the image. But what we're interested in right now is the command in the black box towards the top right. Assuming you have `installed Docker <https://docs.docker.com/desktop/linux/install/>`__ already -- now is a good time to do that -- this is the command used to pull the image from Docker Hub's servers into your computer, allowing you to execute it locally.
```

Comment:
Use an OS-agnostic install link: https://docs.docker.com/desktop/#download-and-install